liblangutil/ErrorReporter.cpp

```diff
@@ -28,6 +28,35 @@ using namespace std;
 using namespace dev;
 using namespace langutil;
 
+namespace langutil {
+
+std::string quote(std::string _name, bool _withSpaces)
+{
+	if (_withSpaces)
+		return " \"" + _name + "\" ";
+
+	return "\"" + _name + "\"";
+}
+
+}
+
+void checkMessage(string const _message)
```

Comment:
Why do you pass it by value and then don't move below?

Reply:
should be by reference. Had a too long break of programming ;)